client/pages/AssessmentSection/DataTable/Chart/chartContainer.tsx

```diff
@@ -30,17 +30,19 @@ const toObject = (x) => {
   Object.entries(x).forEach(([countryIso, countryValues]) => {
     Object.entries(countryValues).forEach(([section, sectionValues]) => {
       Object.entries(sectionValues).forEach(([year, yearValues]: any[]) => {
+        // Display chart for section 1a and 1b
+        if (!['extentOfForest', 'forestCharacteristics'].includes(section)) return
```

Comment:
why hardcode this?

Reply:
Do we have information in metadata which should show chart?

Reply:
Maybe `table.props.odp` = show chart?